spec/support/shared_examples/create_conversion_project_form.rb

```diff
@@ -204,6 +204,16 @@
         expect(form).to be_invalid
       end
     end
+
+    context "when there is another in-progress project with the same urn" do
+      it "is invalid" do
```

Comment:
Can we add another test just to check for the presence of the expected error message?

Reply:
@lozette added that coverage and a tweak. 👍